plutus-wallet-api/ledger/Ledger/Index.hs

```diff
@@ -264,6 +268,20 @@ checkPositiveValues t =
     then pure ()
     else throwError $ NegativeValue t
 
+-- | Minimum transaction fee.
+minFee :: a -> Ada.Ada
```

Comment:
Even if this currently always returns 0 it should have sensible type signature - at least `Set TxIn` and `[TxOut]` of the transaction, probably also the validity interval since the fee may depend on the time

Reply:
I updated the `a` to be `Tx`.